source/includes/steps-fle-create-data-encryption-key.yaml

```diff
@@ -291,8 +357,46 @@ content: |
              status: 0,
              masterKey: { provider: 'local' }
            }
+     .. tab::
+        :tabid: python
+
+        .. code-block:: python
+
+           connection_string = "mongodb://localhost:27017"
+           key_vault_namespace = "encryption.__keyvault"
+
+           fle_opts = AutoEncryptionOpts(
+             kms_providers,
+             key_vault_namespace,
+             mongocryptd_bypass_spawn = True
+           )
+           client = MongoClient(
+             connection_string,
+             auto_encryption_opts = fle_opts
+           )
+           db = client["encryption"]
+           collection = db["__keyVault"]
+           key = collection.find_one({"_id": data_key})
+           print(key)
```

Comment:
see below comment, may want to use pprint

Reply:
Fixed:

* added pprint import & usage